roles/lib_openshift/src/class/oadm_certificate_authority.py

```diff
@@ -0,0 +1,110 @@
+# pylint: skip-file
+
+class CertificateAuthorityConfig(OpenShiftCLIConfig):
+    ''' CertificateAuthorityConfig is a DTO for the oadm ca command '''
```

Comment:
I know there have recently been changes to the cli tooling with deprecating and replacing commands related to CA management. I suspect @abutcher can probably cover those.